Composer/packages/server/src/models/bot/botProject.ts

```diff
@@ -3,9 +3,9 @@
 
 import fs from 'fs';
 
-import isEqual from 'lodash/isEqual';
 import { FileInfo, DialogInfo, LgFile, LuFile, getNewDesigner } from '@bfc/shared';
-import { dialogIndexer, luIndexer, lgIndexer } from '@bfc/indexers';
+import { dialogIndexer, lgIndexer } from '@bfc/indexers';
+import { luIndexer } from '@bfc/indexers/lib/luIndexer';
```

Comment:
Why does this need to be imported via lib, and the others just through the package root?

Reply:
Hi, @a-b-r-o-w-n , the ludown use `#!/usr/bin/env node` in the code. It can't be used in the client. so I separate them. In #1608 , I will convert ludown to bf-lu. This will solve this behavior.